### ISSUE TITLE
tf_ops Documentation request

### ISSUE THREAD

**Issue by null-id** [tensorflow/models#9330]
# Prerequisites

Please answer the following question for yourself before submitting an issue.

- [x] I checked to make sure that this issue has not been filed already.

## 1. The entire URL of the documentation with the issue

https://github.com/tensorflow/models/tree/master/research/sequence_projection

## 2. Describe the issue

Hi, @thunderfyc @expander-robot . I am really interested in your work PRADO. However, I am confused about the tf\_ops part, especially for the sequence\_string\_projection [here](https://github.com/tensorflow/models/blob/master/research/sequence_projection/prado/prado_model.py#L89). Is there any documentation available, since I notice the code is in C++ instead of Python.

**Comment by null-id** [comment]
Hi, is there any help?

**Comment by thunderfyc** [comment]
We wrote a custom TF kernel in C++, and the line you pointed calls that op: https://github.com/tensorflow/models/blob/master/research/sequence_projection/tf_ops/sequence_string_projection.cc

The algorithm behind can be explained in this paper: https://www.aclweb.org/anthology/D19-1506/

**Comment by null-id** [comment]
Yeah, thank you for you reply. However, I mean is there any documentation available for users like me to use the interfaces without reading the source code of the custom kernel?

**Comment by thunderfyc** [comment]
TF api docs are created from python documents. 
We are working on prado keras layer, which may make it easier to follow.
We may also provide a colab later to better demonstrate how to use it

**Comment by null-id** [comment]
Thank you! Looking forward to it!
